cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -20,7 +23,8 @@
 pytestmark = [
     pytest.mark.tier(2),
     pytest.mark.ignore_stream('5.8'),
-    pytest.mark.provider([VMwareProvider, RHEVMProvider],
+    pytest.mark.provider([VMwareProvider, RHEVMProvider, AzureProvider, GCEProvider,
```

Comment:
we can do pytest.mark.provider([CloudInfraProvider]) 
if it supports all cloud and Infra providers.

Reply:
It will also collect OpenStack provider.